tests/integration/conftest.py

```diff
@@ -631,56 +755,30 @@ def _unset_env_var(envars):
     return _unset_env_var
 
 
-# TODO remove when https://issues.redhat.com/browse/RHELC-1389 resolved
-@pytest.fixture(autouse=True, scope="function")
-def remediation_out_of_date_packages(shell):
-    """
-    Remediation fixture.
-    There is an open issue https://issues.redhat.com/browse/RHELC-1389
-    The python3-syspurpose package is left outdated on the system in some cases,
-    causing subsequent tests to fail.
-    Update the package at the end of each test function if needed.
+def get_full_kernel_title(shell, kernel=None):
     """
-    problematic_packages = ["python3-syspurpose"]
-
-    yield
-
-    for package in problematic_packages:
-        shell(f"yum update -y {package}")
-
+    Helper function.
+    Get the full kernel boot entry title.
+    :param kernel: kernel pacakge VRA (version-release.architecture)
+    :type kernel: str
+    :param shell: Live shell fixture
 
-def _create_old_repo(distro: str, repo_name: str):
+    :return: The full boot entry title for the given kernel.
+    :rtype: str
     """
-    Create a repo on system with content that is older than the latest released version.
-    The intended use is for Rocky-8 and Alma-8.
-    """
-    baseurl = None
-    if distro == "alma":
-        baseurl = "https://repo.almalinux.org/vault/8.8/BaseOS/$basearch/os/"
-        if "8.8" in SYSTEM_RELEASE_ENV:
-            baseurl = "https://repo.almalinux.org/vault/8.6/BaseOS/$basearch/os/"
-    elif distro == "rocky":
-        baseurl = "https://download.rockylinux.org/vault/rocky/8.8/BaseOS/$basearch/os/"
-        if "8.8" in SYSTEM_RELEASE_ENV:
-            baseurl = "https://download.rockylinux.org/vault/rocky/8.6/BaseOS/$basearch/os/"
-    else:
-        pytest.fail(f"Unsupported distro ({distro}) provided.")
-    with open(f"/etc/yum.repos.d/{repo_name}.repo", "w") as f:
-        f.write(f"[{repo_name}]\n")
-        f.write(f"name={repo_name}\n")
-        f.write(f"baseurl={baseurl}\n")
-        f.write("enabled=0\n")
-        f.write("gpgcheck=0\n")
+    # Get the full name of the kernel (ignore rescue kernels)
```

Comment:
we could probably do something like:
```
if not kernel:
  return
```
?

Reply:
Maybe rather `raise TypeError` or something like that?

Reply:
I think `ValueError` would be better here. https://docs.python.org/3/library/exceptions.html#TypeError - 

"Passing arguments of the wrong type (e.g. passing a [list](https://docs.python.org/3/library/stdtypes.html#list) 
when an [int](https://docs.python.org/3/library/functions.html#int) is expected) should result in a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError), but passing arguments with the wrong value (e.g. a number outside expected boundaries) should result in a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError)."